esp/services/ws_workunits/ws_workunitsAuditLogs.cpp

```diff
@@ -1444,6 +1444,9 @@ void CWsWorkunitsSoapBindingEx::createAndDownloadWUZAPFile(IEspContext& context,
 
     //CWsWuFileHelper may need ESP's <Directories> settings to locate log files. 
     CWsWuFileHelper helper(directories);
+#ifdef _CONTAINERIZED
+    helper.setRemoteLogAccess(wswService->queryRemoteLogAccessorPtr());
```

Comment:
@jakesmith @rpastrana If I understand the comment correctly, I should roll back all of the changes in https://github.com/hpcc-systems/HPCC-Platform/pull/16343/commits/64e9ea37773bd6b9946802b739544e57ee07c4bd. And remove the LOG line and the OWARNLOG line from initWULogReader() for now. When we revise the Singleton (https://track.hpccsystems.com/browse/HPCC-27983), we may handle the config changes, add the tracing, return a pointer, etc. within the Singleton.

Reply:
Requirements for HPCC-27983 go beyond the requirements for handling a single initialization and ptr return from queryRemoteLogAccessor(). HPCC-27983 is to auto-detect configuration changes and to adjust (if necessary) the underlying object that queryRemoteLogAccessor() returns.

Reply:
I've pushed a pending change to allow the Singleton helper to cope with being initialized (once) to null: https://track.hpccsystems.com/browse/HPCC-27988

Reply:
> And remove the LOG line and the OWARNLOG line from initWULogReader() for now.

I would be happy with that in the short term if @rpastrana - not sure the logging is _that_ useful that we'd miss it, but it's also fine/better to be issued once only (as long as it doesn't complicate all the code that uses it)

Once HPCC-27988 is merged, I'd suggest queryRemoteLogAccessor is changed to return a ptr (and it is valid for it return null), and it can log 1st time initialization and/or exceptions before returning.

That change can be made ahead of work for HPCC-27983 - which is going to be a bit more work.

Reply:
then let's wait for HPCC-27988 to be merged, and have the factory method (which will then only be called once) issue the logging so it's universal.

Reply:
@wangkx https://github.com/hpcc-systems/HPCC-Platform/pull/16368 (HPCC-27988) is now merged.